test/langtools/ProblemList.txt

```diff
@@ -71,3 +71,6 @@ tools/sjavac/ClasspathDependencies.java
 ###########################################################################
 #
 # jdeps
+
+
+jdk/jshell/ToolTabSnippetTest.java 1234567 generic-all
```

Comment:
This still fails?

Reply:
Yeah, intermittendly, in GHA. It's not related to this change, but I wanted GHA green.